Automattic/wp-calypso
# Login Assistance: Cleanup the migrate CTA on failed login experiment

Related to #90950

## Proposed Changes

* Revert the experiment to show a migrate CTA when login fails.

## Why are these changes being made?


* It was decided to stop the experiment because it was causing a negative impact on the login complete metric. More info about the decision can be found here: pdxWSz-1yz-p2

## Testing Instructions

* Go to http://calypso.localhost:3000/log-in.
* Enter a non-existing email.
* Make sure you see a "User does not exist" error and don't see the migrate CTA message.
* Make sure you can log in as expected with a real account.

## Pre-merge Checklist
